plugins/hydra_optuna_sweeper/example/sphere.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from typing import Any
+
+import hydra
+from omegaconf import DictConfig
+
+logger = logging.getLogger(__name__)
+
+
+@hydra.main(config_path="conf", config_name="config")
+def sphere(cfg: DictConfig) -> Any:
```

Comment:
Could we add a tighter type for return

Reply:
Thank you for your suggestion. I made it `float` in https://github.com/facebookresearch/hydra/pull/1132/commits/abd74d9606293572a2eea83a2291b4342dac66b4.